x-pack/plugins/apm/public/components/shared/transaction_action_menu/sections.ts

```diff
@@ -204,6 +208,28 @@ export const getSections = ({
     },
   ];
 
+  const serviceMapHref = apmRouter.link('/service-map', {
+    query: {
+      rangeFrom: urlParams.rangeFrom ?? 'now-15m',
+      rangeTo: urlParams.rangeTo ?? 'now',
```

Comment:
I think `rangeFrom` and `rangeTo` are required at this point. Perhaps you need to convert `useLegacyUrlParams` to using the new router methods for getting the route params

```suggestion
      rangeFrom: urlParams.rangeFrom,
      rangeTo: urlParams.rangeTo,
```

Reply:
Yeah it's about time this get updated.